### PR TITLE
Feat/pie ios

### PR DESCRIPTION
**Pull request by trevorwhealy**

### CODE REVIEW

**Comment by donovanCarreno** [comment]
🚢

File: mobile/t2/ios/t2-Bridging-Header.h [hunk 1]
```diff
@@ -0,0 +1,4 @@
+//
+//  Use this file to import your target's public headers that you would like to expose to Swift.
+//
+
```
Comment (danejordan) [inline mobile/t2/ios/t2-Bridging-Header.h:4]: an empty bridging header file?

Comment (trevorwhealy) [inline mobile/t2/ios/t2-Bridging-Header.h:4]: /giphy _shrugs_

**Comment by donovanCarreno** [comment]
old stuff

**Pull request closed without merging**